docs/web-vitals.md

```diff
@@ -0,0 +1,362 @@
+# Web Vitals
+
+By default, Honeycomb includes events for all web vitals except first-input-delay. First-input-delay is set to be replaced by interaction-to-next-paint as a core web vital in March 2024.
+
+## Metric Attributes
```

Comment:
This might sound pedantic, but I don't _think_ it is, or at least I don't intend it to be. It's because of an unfortunate collision between existing naming/usage and our desired usage.... Anywhere we can avoid using the term "metric" and replace it with "vital" I think that would helpful, to distinguish that we are not using the metric signal here.

Reply:
no, i love it — definitely not pedantic, +++ clarity